src/workerd/util/sqlite.c++

```diff
@@ -236,6 +236,37 @@ static constexpr kj::StringPtr ALLOWED_SQLITE_FUNCTIONS[] = {
   "json_tree"_kj,
 };
 
+// https://www.sqlite.org/pragma.html
+static constexpr kj::StringPtr ALLOWED_READ_PRAGMAS[] = {
+  // We allowlist these SQLite pragmas (for read only, never with arguments).
+  "data_version"_kj,
+  "page_count"_kj,
+  "freelist_count"_kj,
+
+  // Let a user see some internal stats?
+  "max_page_count"_kj,
+  "page_size"_kj,
+};
+
+static constexpr kj::StringPtr ALLOWED_WRITE_PRAGMAS[] = {
+  // We allowlist some SQLite pragmas for changing internal state
+
+  // Toggle constraints on/off
```

Comment:
I'm OK with this set of pragmas as they seem like logical options for an application to be able to toggle. But, can we specifically validate that the argument is a boolean?